src/accordion/pup-e2e.js

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+/*eslint-env node*/
+
+describe('Google', () => {
+  beforeAll(async () => {
+    await page.goto('https://google.com');
+  });
+
+  it('should display "google" text on page', async () => {
+    await expect(page).toMatch('google');
```

Comment:
`expect(page)` works here? Isn't that a puppeteer page object? Just wondering if we need to get the content from the page.